### PR TITLE
fix(core,platform): feedinput component style cleanup

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/5343

#### Please provide a brief summary of this pull request.

removing the feedinput component style cleanup

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,8 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.5.0",
-    "fundamental-styles": "0.19.0-rc.38",
+    "fundamental-styles": "0.19.0-rc.42",
+    "hammerjs": "2.0.8",
```
Comment (InnaAtanasova) [inline package.json:120]: Why do you bring `hammerjs`? @N1XUS recently created a PR to remove it.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Apart from `hammerjs` dependency the rest looks ok

File: package.json (hunk 1 above)
Comment (Lokanathan-k) [reply package.json]: rebased to the latest changes

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by puru-hk** [comment]
Looks fine.

**Review by puru-hk — APPROVED** [review]

**Review by N1XUS — APPROVED** [review]

**Pull request merged**